client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -28,6 +27,19 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 		onVideoLoaded: noop,
 	};
 
+	constructor( props ) {
+		super( props );
+
+		this.enableAutoplay = props.isPlaying;
+	}
+
+	// We use a class property because we only want to set this once
+	// before the component got rendered for the first time. `isPlaying`
+	// could change over time but it doesn't make sense to re-set the
+	// `autoPlay` attribute for the videopress iframe (which would also
+	// cause the iframe to flicker).
+	enableAutoplay = false;
```

Comment:
Is this line actually necessary? Won't we always overwrite this default value before mounting, when `constructor()` is called? If you agree, feel free to move this comment just above L33.

Reply:
I think you're right 👍🏻 will add a fix.